lib/utils.js

```diff
@@ -61,33 +61,36 @@ exports.extract = function extract(selectorText) {
  */
 
 exports.parseCSS = function(css) {
-  var rules = cssom.parse(css).cssRules || [];
+  var parsed = mensch.parse(css);
+  var rules = typeof parsed.stylesheet != 'undefined' && parsed.stylesheet.rules ? parsed.stylesheet.rules : [];
   var ret = [];
 
   for (var i = 0, l = rules.length; i < l; i++) {
-    if (rules[i].selectorText) { // media queries don't have selectorText
+    if (rules[i].type == 'rule') {
       var rule = rules[i];
-      var selectors = exports.extract(rule.selectorText);
+      var selectors = rule.selectors;
 
       for (var ii = 0, ll = selectors.length; ii < ll; ii++) {
-        ret.push([selectors[ii], rule.style]);
+        ret.push([selectors[ii], rule.declarations]);
       }
     }
   }
 
   return ret;
 };
 
-
-var getStringifiedStyles = function(rule) {
-  var styles = [];
-  for (var style = 0; style < rule.style.length; style++) {
-    var property = rule.style[style];
-    var value = rule.style[property];
-    var important = rule.style._importants[property] ? ' !important' : '';
-    styles.push('    ' + property + ': ' + value + important + ';');
-  }
-  return styles;
+var removeStyle = function(style, startPos, endPos, skipRows, startOffset, endOffset, insert) {
+  var styleRows = style.split("\n");
+  var start = startOffset;
+  var end = endOffset;
+  for (var r = 1 + skipRows; r < startPos.line; r++) start += styleRows[r - 1 - skipRows].length + 1;
+  start += startPos.col;
+  if (endPos !== null) {
+    for (var r2 = 1 + skipRows; r2 < endPos.line; r2++) end += styleRows[r2 - 1 - skipRows].length + 1;
+    end += endPos.col;
+  } else end += style.length + 1;
```

Comment:
Please include block style braces `{` and `}` for these `else` and `for` statements. I think the JSCS rules in the repo require that.